src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Object/KeyValuePairConverter.cs

```diff
@@ -79,7 +77,6 @@ internal override void Initialize(JsonSerializerOptions options, JsonTypeInfo? j
             }
```

Comment:
Can't comment on the actual line, but is a `TryLookupConstructorParameter` override still needed here? I would assume not given we're adopting same semantics as the base class?

Reply:
When I remove the override the compiler throws exceptions "Field 'ArgumentState.FoundValue' is never assigned to, and will always have its default value false" which is weird. Do you know how to get rid of the check?

Reply:
Huh, where does that error occur if you're deleting the method?

Reply:
It is a static code analysis error that occurs during building. I found a workaround, would be nice to know your opinion about that.